packages/dx-react-chart/docs/guides/stacked-series.md

```diff
@@ -0,0 +1,19 @@
+# React Chart - Stacked Series
+
+The Chart component supports stacked series. Use `Stack` plugin and property `stack` in the series to draw points side-by-side or on top of each other, customize order and offset of the series in stacks.
+
+## Basic Setup
+
+To draw several series side-by-side you should only use `Stack` plugin. The following example demonstrates drawing several bar series:
+
+.embedded-demo({ "path": "chart-basic/band-axis", "showThemeSelector": true })
+
+To draw several series in one stack assign the same value to the `stack` property of series. If you don't want to draw a series in the stack, for example, a line series, set `stack` property to null.
```

Comment:
If you don't want to draw a series in the stack.
May be - "If you don't want to allow a series in the calculation of the stack", better?